### PR TITLE
feat(fe2): app authorization workflow redesign [WBX-217]

### PR DESCRIPTION
**Pull request by fabis94**

### CODE REVIEW

File: packages/frontend-2/pages/developer-settings.vue [hunk 1]
```diff
@@ -220,18 +280,27 @@ import {
...
 } from '~~/lib/developer-settings/helpers/types'
 import {
   developerSettingsAccessTokensQuery,
-  developerSettingsApplicationsQuery
+  developerSettingsApplicationsQuery,
+  developerSettingsAuthorizedAppsQuery
 } from '~~/lib/developer-settings/graphql/queries'
 import { useQuery } from '@vue/apollo-composable'
 
+// TODO: Hide first party apps like in FE1
```
Comment (andrewwallacespeckle) [inline packages/frontend-2/pages/developer-settings.vue:298]: Just checking this is something for future, and not something you meant to come back to

**Review by andrewwallacespeckle — COMMENTED** [review]
This looks all good. Although I wasn't able to trigger the new auth screen. Maybe you could show me how to do this on Monday?

**Comment by fabis94** [comment]
@andrewwallacespeckle You can test out authorization for the Explorer app,. if you like. Open localhost:3000/explorer locally.

**Comment by andrewwallacespeckle** [comment]
> @andrewwallacespeckle You can test out authorization for the Explorer app,. if you like. Open localhost:3000/explorer locally.

Hmm, this didn't show the new Authorise dialog, once logged in, it just opened the explorer. Am I doing something wrong?

**Comment by fabis94** [comment]
@andrewwallacespeckle You're probably already authorized. Or its the fact that it has trustByDefault on, and it auto approves when you open the route. I used to just change this value in the DB directly to go around that, but without a DB client I'm not sure what's the best way for you to test this. Probably by creating a new app in the developer settings and then trying to auth against that.

**Pull request merged**